public/admin/templates/grid.tmpl.html

```diff
@@ -91,4 +127,12 @@ <h2>Grid Rows</h2>
     	allowEmpty:true,
     	showButtons: false
     });
+    
+    function showColorOptions() {
```

Comment:
Can these be angula (ished) ?

Reply:
Agree using the angular way of updating this is better than by directly modifying the DOM